RetailCoder.VBE/UI/Controls/DeclarationTypeToStringConverter.cs

```diff
@@ -9,14 +9,13 @@ public class DeclarationTypeToStringConverter : IValueConverter
     {
         public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
         {
-            if (value is DeclarationType)
+            if (!(value is DeclarationType type))
```

Comment:
Yuck. Don't invert conditions needlessly. It doesn't reduce nesting enough to talk about, for all R#'s shooting it's mouth off.